pallets/runtime/src/bridge.rs

```diff
@@ -311,7 +318,8 @@ decl_module! {
             DispatchResult
         {
             let sender = ensure_signed(origin)?;
-            ensure!(sender == Self::controller(), Error::<T>::BadCaller);
+            ensure!(sender == Self::controller() || sender == Self::admin(), Error::<T>::BadCaller);
```

Comment:
isn't it better to add a TODO so we can remove the admin check at the production?

Reply:
added todo comment